### DISCUSSION TITLE
Adding Anime4k to Plex HTPC on Linux (Flatpak)

### DISCUSSION THREAD

**Discussion by MichaelWangenstein** [bloc97/Anime4K#207] [General]
I wasn't able to find any documentation on this so I'm adding it here in case anyone comes looking for it in the future.

1. Download the template files located [here](https://github.com/bloc97/Anime4K/blob/master/md/GLSL_Instructions_Linux.md).
2. Make any edits that you need to mpv.conf and input.conf.
3. Navigate to: /home/.var/app/tv.plex.PlexHTPC/data/plex/
4. Copy the files into the directory.
5. Launch Plex HTPC and check your results.

**Comment by Tama47** [comment]
Thanks! I added this to the pull request https://github.com/bloc97/Anime4K/pull/225.

[GLSL_Instructions_Linux_Plex.md](https://github.com/Tama47/Anime4K/blob/master/md/GLSL_Instructions_Linux_Plex.md)

Let me know if anything need improving. 
Feel free for anyone to add more instructions for other distros.
